bzt/modules/apiritif/generator.py

```diff
@@ -954,6 +955,9 @@ def _gen_webdriver(self):
                 value=ast_call(
                     func=ast_attr("webdriver.%s" % browser))))  # todo bring 'browser' to correct case
 
+        if self.OPTIONS in self.executor.settings:
+            body.append(self._get_selenium_options(browser))
```

Comment:
i see that sometimes we use `body.append` and  `body.extend`. 
I think you should use `extend`, because this function returns array